pkg/ldapserver/server.go

```diff
@@ -124,6 +124,7 @@ func (server *Server) Serve(ln net.Listener) error {
 				}
 				break
 			}
+			log.Printf("New Connection on %s", ln.Addr())
```

Comment:
Forgotten debug?  This should probably not be logged in normal operations.

Reply:
Actually, this only logged at Debug level. As the all the other messages coming from the standard go logger. See: https://github.com/libregraph/idm/blob/master/server/server.go#L113 , that's why I kept it here.

Would you still prefer to get it removed?

Reply:
Got it - thats fine then :+1: